blobfuse/directoryapis.cpp

```diff
@@ -228,13 +228,26 @@ int azs_rmdir(const char *path)
 int azs_statfs(const char *path, struct statvfs *stbuf)
 {
     std::string pathString(path);
-    const char * mntPath;
-    std::string mntPathString = prepend_mnt_path_string(pathString);
-    mntPath = mntPathString.c_str();
 
-    int res = statvfs(mntPath, stbuf);
-    if (res == -1)
-        return -errno;
+    int dir_result = is_directory_empty(str_options.containerName, "/", pathString.substr(1));
+    bool path_exists = false;
```

Comment:
Can we simplify and just call azs_get_attr to determine if it's a valid file or directory?

Reply:
Ok I can do that